# WILDFIRE RISK AND SEVERITY ANALYSIS

## Project Objective:
To explores environmental conditions that influence wildfire occurrence and severity. 

### Project Goal:
Understand how temperature, humidity, wind, and rainfall affect wildfire risk. 
Identify patterns that can support better wildfire preparedness and response.

## Import Libraries

In [13]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

## Loading And Preview Of The Dataset 

In [15]:
df = pd.read_csv("../data/forestfires.csv")
df.head()

,X,Y,month,day,FFMC,DMC,DC,ISI,temp,RH,wind,rain,area
0,7,5,mar,fri,86.2,26.2,94.3,5.1,8.2,51,6.7,0.0,0.0
1,7,4,oct,tue,90.6,35.4,669.1,6.7,18.0,33,0.9,0.0,0.0
2,7,4,oct,sat,90.6,43.7,686.9,6.7,14.6,33,1.3,0.0,0.0
3,8,6,mar,fri,91.7,33.3,77.5,9.0,8.3,97,4.0,0.2,0.0
4,8,6,mar,sun,89.3,51.3,102.2,9.6,11.4,99,1.8,0.0,0.0


In [14]:
df.shape

(517, 13)

## Dataset Structure

In [16]:
df.dtypes

X          int64
Y          int64
month     object
day       object
FFMC     float64
DMC      float64
DC       float64
ISI      float64
temp     float64
RH         int64
wind     float64
rain     float64
area     float64
dtype: object

## Checking For Missing Values

In [18]:
df.isnull().sum()

X        0
Y        0
month    0
day      0
FFMC     0
DMC      0
DC       0
ISI      0
temp     0
RH       0
wind     0
rain     0
area     0
dtype: int64

## Understanding Variables In The Dataset

| Column | Meaning |
|---|---|
| X | x-axis spatial coordinate of the forest fire location |
| Y | y-axis spatial coordinate of the forest fire location |
| month | month the fire incident occurred |
| day | day of the week the fire incident occurred |
| FFMC | Fine Fuel Moisture Code |
| DMC | Duff Moisture Code |
| DC | Drought Code |
| ISI | Initial Spread Index |
| temp | temperature in °C |
| RH | relative humidity (%) |
| wind | wind speed |
| rain | rainfall amount |
| area | burned forest area |

## Data Overview

The dataset has 517 rows and 13 columns. It includes both categorical and numerical variables related to forest fire conditions, weather, and burned area.The dataset appears structured and suitable for analysis.

## Initial Observations

- The dataset is clean and well-structured.
- No missing values were found.
- Most variables are numerical.
- 'month' and 'day' are categorical variables.
- 'area' appears to be the main variable of interest for analysis.